bitnami/memcached/Chart.yaml

```
@@ -22,4 +22,4 @@ name: memcached
 sources:
   - https://github.com/bitnami/bitnami-docker-memcached
   - http://memcached.org/
-version: 5.13.7
+version: 5.13.8
```

We're adding a new feature, therefore we should bump the minor version:
```suggestion
version: 5.14.0
```